### ISSUE TITLE
Time info text leafs on timeline have insufficient color contrast ratio (3:22:1)

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4569]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 1.4.3 Contrast (Minimum)

**Test Environment:**
**Test URL** http://cx-guidelines.vmware.com/clarity3FullTesting/
**OS** Tested on Windows but issue is independent of platform.
**User Agents/Assistive Technologies** Firefox 76.0

**Description:**
Time info text leaves on horizontal or vertical timeline are having insufficient color contrast ratio i.e. it is 3.22:1. Due to this issue, low vision users might find it difficult to read the time info details.

**Steps to Reproduce**
Go to horizontal or vertical timeline and check the color contrast ratio provided for the time info text leaf i.e. the text &quot;11.59 am&quot;.

To meet the AA conformance level the color contrast ratio should be at least 4.5:1 for the text size less than 14pt bold or 18pt. For text size equal to or greater than 14pt bold or 18pt, color contrast ratio can be at least 3:1.
